camel/societies/role_playing.py

```diff
@@ -514,8 +532,158 @@ async def ainit_chat(
             content=init_msg_content,
         )
 
+        # Reset conversation turn counter
+        self._conversation_turn = 0
+
         return init_msg
 
+    def _create_checkpoint(self, checkpoint_name: str) -> None:
+        r"""Create a memory checkpoint by saving the current state of both
+            agents' memories.
```

Comment:
docstring format, same for other part

```suggestion
        r"""Create a memory checkpoint by saving the current state of both
        agents' memories.
```